### DISCUSSION TITLE
Programs where fullscreen does not work

### DISCUSSION THREAD

**Discussion by marticliment** [martinet101/ElevenClock#191] [Q&A] [answered]
Please post here the name of the program and it's process name if Elevenclock is not hiding on fullscreen as intended

**Comment by stukindaguy** [comment, marked as answer]
Microsoft's Standalone Windows RDP Client  "Remote Desktop"

Process name for the main launcher window: msrdcw.exe
Process name for the actual RDP session (this window would be the fullscreen one): msrdc.exe

Elevenclock 2.4 not hiding as expected. Using Remote Desktop version 1.2.2459.0.

**Reply by marticliment** [reply]
Aolved, coming in 2.5

**Reply by Enzoplobeast** [reply]
is this fixed yet, I have this same issue

**Reply by Enzoplobeast** [reply]
my remote desktop process is called mstsc.exe

**Reply by stukindaguy** [reply]
Issue is resolved for me as of 2.5.
